Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -281,6 +281,25 @@ private void OnFocusChanged(bool focus)
         public Vector2 screenSize => new Vector2(Screen.width, Screen.height);
         public ScreenOrientation screenOrientation => Screen.orientation;
 
+        public bool normalizeScrollWheelDelta
+        {
+#if UNITY_6000_0_OR_NEWER
+            get => NativeInputSystem.normalizeScrollWheelDelta;
+            set => NativeInputSystem.normalizeScrollWheelDelta = value;
+#else
+            get; set;
+#endif
+        }
+
+        public float scrollWheelDeltaPerTick
+        {
+#if UNITY_6000_0_OR_NEWER
+            get { return NativeInputSystem.GetScrollWheelDeltaPerTick(); }
```

Comment:
Recommend being consistent with getter below and use the short-hand version with =>